examples/misc/lib/effective_dart/design_good.dart

```diff
@@ -36,28 +36,37 @@ void miscDeclAnalyzedButNotTested() {
   (Func1 entryPoint, message, Iterable elements, String pattern) {
     // #docregion omit-verb-for-bool-param
     Isolate.spawn(entryPoint, message, paused: false);
-    new List.from(elements, growable: true);
-    new RegExp(pattern, caseSensitive: false);
+    var copy = new List.from(elements, growable: true);
+    var regExp = new RegExp(pattern, caseSensitive: false);
     // #enddocregion omit-verb-for-bool-param
   };
 
   (Queue queue, window, connection) {
     // #docregion verb-for-func-with-side-effect
-    list.add(ellipsis);
+    list.add("element");
     queue.removeFirst();
     window.refresh();
-    connection.downloadData();
     // #enddocregion verb-for-func-with-side-effect
   };
 
   (bool Function(dynamic) test) {
     // #docregion noun-for-func-returning-value
-    list.elementAt(3);
-    list.firstWhere(test);
-    string.codeUnitAt(4);
+    var element = list.elementAt(3);
+    var first = list.firstWhere(test);
+    var char = string.codeUnitAt(4);
     // #enddocregion noun-for-func-returning-value
   };
 
+  (bool Function(dynamic) test) {
```

Comment:
This anonymous function doesn't need any parameters (`test` isn't used in the body).
On the other hand, you could make `database` and `packageGraph` parameters.

Reply:
Done.